src/pf-utils/pf-utils.js

```diff
@@ -38,4 +38,76 @@ let I18nUtil = function() {
     self.mixin = mixin;
   };
 };
-export let i18n = new I18nUtil();
+let i18n = new I18nUtil();
+export {i18n};
+
+/**
+ * --------------------------------------------------------------------------
+ * PfUtil
+ * Native Javascript for Bootstrap 3 | Internal Utility Functions
+ * --------------------------------------------------------------------------
+ */
+
+class PfUtil {
```

Comment:
Can this be renamed something more specific (e.g., NativeUtil or NativeBootstrapUtil)? (Not certain this code is dependent on Bootstrap?) The intention was to add specific objects like i18nUtil, but not PfUtil itself which kind of becomes a catch-all for random functions.

Alternatively, we might want to move each object to its own file?